docs/data/data-grid/clipboard/clipboard.md

```diff
@@ -0,0 +1,99 @@
+# Data Grid - Copy and paste
+
+<p class="description">Copy and paste data using clipboard.</p>
+
+## Clipboard copy
+
+You can copy selected grid data to the clipboard using the <kbd class="key">Ctrl</kbd>+<kbd class="key">C</kbd> (<kbd class="key">⌘ Command</kbd>+<kbd class="key">C</kbd> on macOS) keyboard shortcut.
+The copied cell values are separated by a tab (`\t`) character and the rows are separated by a new line (`\n`) character.
+
+The priority of the data copied to the clipboard is the following, from highest to lowest:
+
+1. If more than one cell is selected (see [Cell selection<span class="plan-premium" title="Premium plan"></span>](/x/react-data-grid/cell-selection/)), the selected cells are copied
+2. If one or more rows are selected (see [Row selection](/x/react-data-grid/row-selection/)), the selected rows are copied
+3. If there is a single cell selected, the single cell is copied
+
+{{"demo": "ClipboardCopy.js", "bg": "inline", "defaultCodeOpen": false}}
+
+## Clipboard paste [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan)
+
+:::warning
+This feature is experimental, it needs to be explicitly activated using the `clipboardPaste` experimental feature flag.
+
+Additionally, to make sure the copied cells are formatted correctly and can be parsed,
+it's recommended to set the `unstable_ignoreValueFormatterDuringExport` prop to `true`.
+
+```tsx
+<DataGridPremium
+  experimentalFeatures={{ clipboardPaste: true }}
+  unstable_ignoreValueFormatterDuringExport
+/>
+```
+
+:::
+
+You can paste data from clipboard using the <kbd class="key">Ctrl</kbd>+<kbd class="key">V</kbd> (<kbd class="key">⌘ Command</kbd>+<kbd class="key">V</kbd> on macOS) keyboard shortcut.
+The paste operation only affects cells in the [columns that are `editable`](/x/react-data-grid/editing/#making-a-column-editable).
+
+Same as with editing, you can use `valueParser` to modify the pasted value and `valueSetter` to update the row with new values.
+See [Value parser and value setter](/x/react-data-grid/editing/#value-parser-and-value-setter) editing section for more details.
+
+The behavior of the clipboard paste operation depends on the selection state of the data grid and the data pasted from clipboard.
+The priority is the following, from highest to lowest:
```

Comment:
The way I understand the following table, it's a matrix combining "copy from" with "paste to" scenarios.
Does it still make sense to speak in terms of priority?

Reply:
The priority here makes sense because you can have both selected rows **and** selected cell range. In this case, the selected cell range has higher priority and selected rows will be ignored.